src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -164,6 +165,7 @@ const OwnerPanel = ({ seller }: Props) => {
         sellerId={seller.seller_id}
         sellerName={seller.name}
         costPerMeal={costPerMealDollars}
+        nonProfitLocationId={seller.non_profit_location_id}
```

Comment:
Seems ok for now, just be careful to handle the case if this value is nil :)

Reply:
Yup! Just tested with backend on master not having this variable and everything is fine, handling the null cases ok!